package.json

```diff
@@ -894,6 +894,7 @@
     "@typescript-eslint/eslint-plugin": "^5.22.0",
     "@typescript-eslint/parser": "^5.22.0",
     "@vscode/test-electron": "^2.1.3",
+    "mkdirp": "1.0.3",
```

Comment:
I don't understand one thing, if some dependency requires mkdirp in `1.x` then why on earth we need to specify this manually. I mean, come js, why are that dumb :(

Or maybe I'm too used to JVM dependency mechanics..

Reply:
I don't really understand it either, I would think they would use the latest version, but it doesn't :man_shrugging: